src/Symfony/Bundle/SecurityBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.4
+---
+
+ * Register alias for argument for password hasher when the key is not a class name
```

Comment:
Let's be more explicit and maybe give an example. Without any context, I don't understand what it means.

Reply:
@fabpot I added an example, is it okay now?